superset-frontend/babel.config.js

```diff
@@ -16,30 +16,36 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+const packageConfig = require('./package.json');
+
 module.exports = {
   sourceMaps: true,
-  sourceType: 'unambiguous',
+  sourceType: 'module',
```

Comment:
`'module` is preferred as it is more strict. If we can solve the script imports, then there is not need for `unambiguous`.